Model/DataProviderInterface.php

```diff
@@ -43,7 +43,7 @@ public function getItem($id, $fetchData = false);
      * @param int         $page
      * @param array       $filters
      * @param int|null    $itemsPerPage
-     * @param string|null $order
+     * @param array|null  $order
```

Comment:
So a TypeHint must be added to the interface and implementations (`array $order = null`).